software/o_c_REV/vector_osc/waveform_library.h

```diff
@@ -39,16 +57,16 @@ VOSegment library_waveforms[] = {
 
     // Sine
     VOSegment {12, VO_TOC},
-    VOSegment {192, 1},
+    VOSegment {191, 1},
```

Comment:
I obtained 191 and 17 instead of 192 and 18 when rounding a sine wave that runs from 0 to including 255. Therefore, these changes should reduce THD, which I subjectively verified by listening:
`>>> 255 * (1 + np.sin(np.linspace(0, 2 * np.pi, 12, endpoint=False))) / 2`
`array([127.5, 191.25, 237.91823898, 255., 237.91823898, 191.25, `
`       127.5, 63.75, 17.08176102, 0., 17.08176102, 63.75])`